.rubocop.yml

```diff
@@ -6,6 +6,7 @@ require:
 # Enable new cops when we update rubocop to a newer version
 AllCops:
   NewCops: enable
+  TargetRubyVersion: 2.7
```

Comment:
Why not make rubocop enforce Ruby 3 rules right away? 🤔

Reply:
That means we'll have to change the code to use Ruby 3 syntax or API to address the new rubocop issues. I don't think we should use Ruby 3 right now, because the apps are still using 2.7. Once a new release-toolkit is released (which contains all the fixes for the Ruby 3 compatibility), I'll try to upgrade a few apps to use Ruby 3. If all go well, we can then changing this library to [require Ruby 3 and above](https://github.com/wordpress-mobile/release-toolkit/blob/trunk/fastlane-plugin-wpmreleasetoolkit.gemspec#L15). That's when we can drop 2.7 and let rubocop enforce Ruby 3 rules.